cost-analyzer/templates/_helpers.tpl

```diff
@@ -1108,4 +1122,43 @@ SSO enabled flag for nginx configmap
   {{- else -}}
     {{- printf "false" -}}
   {{- end -}}
-{{- end -}}
\ No newline at end of file
+{{- end -}}
+
+{{- define "gcpCloudIntegrationCheck" }}
+  {
+    "gcp":
+      {
+        [
+          {
+              "bigQueryBillingDataDataset": "{{ .Values.kubecostProductConfigs.bigQueryBillingDataDataset }}",
+              "bigQueryBillingDataProject": "{{ .Values.kubecostProductConfigs.bigQueryBillingDataProject }}",
+              "bigQueryBillingDataTable": "{{ .Values.kubecostProductConfigs.bigQueryBillingDataTable }}",
+              "projectID": "{{ .Values.kubecostProductConfigs.projectID }}"
+          }
+        ]
+      }
+  }
+{{- if ((.Values.kubecostProductConfigs).bigQueryBillingDataDataset) }}
+{{- fail "\nKubecost 2.0 requires a change to the method that cloud-provider billing integrations are configured.\nPlease use this output to create a cloud-integration.json config. See:\n<https://docs.kubecost.com/install-and-configure/install/cloud-integration#adding-a-cloud-integration>\nfor more information"}}
+{{- end }}
+{{- end }}
+
+
+{{- define "azureCloudIntegrationCheck" }}
```

Comment:
@chipzoller is there a way to fail and print this out? See the fail note for this and the gcpCloudIntegrationCheck

Reply:
Yes, you can just define a var similar to `cloudIntegrationJSON` which is a block scalar and print this out in the failure message referencing your var.

I'll also note that the current schema for that JSON document is missing `azureContainerPath` and `azureCloud` keys. Seems like they should be in `cloudIntegrationJSON`, right?